pkg/security/podsecuritypolicy/util/util.go

```diff
@@ -182,8 +183,9 @@ func PSPAllowsHostVolumePath(psp *extensions.PodSecurityPolicy, hostPath string)
 		return true
 	}
 
+	path := filepath.Clean(hostPath)
```

Comment:
Yeah, that seems wrong.  I have trouble imagining how that is allowed, or how it would be helpful.